app/controllers/idv/agreement_controller.rb

```diff
@@ -37,6 +37,7 @@ def update
 
         # for the 50/50 state
         flow_session['Idv::Steps::AgreementStep'] = true
+        flow_session['Idv::Steps::InPerson::VerifyStep'] = true
```

Comment:
This added line belongs in InPerson::VerifyInfoController#update, not in AgreementController.

Reply:
resolved with commit 0f3c14c5b0bff64d367b2b40b8ccbfe550035f0f